package.json

```diff
@@ -9,8 +9,7 @@
   "author": "Mozilla",
   "license": "MPL-2.0",
   "devDependencies": {
-    "mustache": "2.3.0",
-    "pioneer-studies-addon-utils": "1.0.1"
```

Comment:
Because we have a `package-lock.json`, using semver versions instead of pinning here is the correct thing to do.